### ISSUE TITLE
Windows 10: Vim with fzf installed in linux subsystem 

### ISSUE THREAD

**Issue by jfun** [junegunn/fzf.vim#186]
- Category
    - [x] Question
    - [ ] Bug
    - [ ] Suggestion
- OS
    - [ ] Linux
    - [ ] Mac OS X
    - [x] Windows
    - [ ] Etc.
- Vim
    - [x] Vim
    - [ ] Neovim

So, I have vim set up in the Windows 10 environment, and fzf installed in the linux subsystem. I can open up a normal command prompt and run `bash -c ~/.fzf/bin/fzf` and get the fzf prompt, and it works exactly as expected. 

Basically my question is where can the run command be changed in the plugin to invoke `bash -c ~/.fzf/bin/fzf` instead? I'm a total vimscript noob, but I think it should just work?

**Comment by junegunn** [comment]
So the Go version of fzf finally runs on Windows thanks to linux subsystem? That's a great news. We've been trying to make it work on Windows for quite some time to no avail (see https://github.com/junegunn/fzf/issues/147).

Anyway since I'm not a Windows user, I don't think I can give you the answer right now. I'll try to test fzf on Windows when I get a chance.

**Comment by jfun** [comment]
Yeah that's correct. I did some quick fiddling around, the hurdle I came across is that vim will provide Windows paths to the fzf#run call, but since fzf is run under Linux, obviously these paths don't quite translate.

**Comment by jmlucjav** [comment]
maybe dummy question but...are you running window's version of vim or the ubuntu one in WSL? If you run both vim and fzf in WSL they should use all linux paths I guess? Of course, that might have other downsides, I am not yet able to try myself.

**Comment by jfun** [comment]
fzf / fzf.vim works perfectly fine in WSL without any changes. The only thing is that I don't have the time to fiddle around and perfect my setup (this is a work setup, I wouldn't be using Windows otherwise). I had already some time faffing about with cmder and using Xming & gvim), neither of which *just work* out of the box.

But yes, if you have the time, just go use vim in WSL and you can enjoy fzf on Windows :-)

**Comment by PhilT** [comment]
Are up and down arrows working for you? fzf bails as soon as I press up or down arrows outputting `[A` or `[B` in the console. This also happens when running fzf from the command line so not a vim related problem.

Works okay with CTRL+J/K so that's workaround for now.
